utils/build-script

```diff
@@ -514,6 +514,14 @@ name of the directory under $SWIFT_BUILD_ROOT where the build products will be
 placed""",
         metavar="PATH")
 
+    parser.add_argument("--cc",
+        help="specify the C compiler to be used",
+        action="store", dest="cc")
```

Comment:
Do we need the `dest=` here?  It seems like it can be inferred, like we do for for `--ios`.